### ISSUE TITLE
[4pt] Update & push .pot files in CI

### ISSUE THREAD

**Issue by NickVolynkin** [tarantool/doc#1866]
Do after #1864 

Daily or after each merge to stable branches, update .pot files and push them to Crowdin.
Update should only be done in the root repository, without submodules (yet).

Commit and push changes to the same branch. It's better to avoid triggering rebuilds on such commits.

Ask @NickVolynkin for an API token for Crowdin. Store it in a vault for GH Actions.

**Comment by artembo** [comment]
Need to set up a workflow with these actions:
- https://github.com/marketplace/actions/add-commit — for autocommit changes
- https://github.com/marketplace/actions/crowdin-action — pull the new translations

**Comment by NickVolynkin** [comment]
Merged the first part, made a separate issue for the second: #1935
